src/status_im2/contexts/quo_preview/community/community_stat.cljs

```diff
@@ -0,0 +1,21 @@
+(ns status-im2.contexts.quo-preview.community.community-stat
+  (:require
+    [quo.core :as quo]
+    [reagent.core :as reagent]
+    [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:key  :value
+    :type :text}
```

Comment:
We have the descriptor type `:number`, which will be just like `:text`, but will parse the input value to an integer before the value is passed over to the component props. Unfortunately the naming is not very precise, I don't why I named it `number` if behind the scenes I used `parseInt`...

https://github.com/status-im/status-mobile/blob/9a5ed62e1ffc931332a75e29c342e41b42d5b2b2/src/status_im2/contexts/quo_preview/preview.cljs#L100

If possible @ajayesivan, I think our previews should better reflect the args that will be passed in production. `utils.money/format-amount` also expects a number, since it performs calculations with it, so it's a good reason to change the preview too.

Malli will help us out :)